online-shop/src/API/CartProducts.tsx

```diff
@@ -0,0 +1,15 @@
+// Initially, the cart is empty
+
+const ProductsInCart = {
+    products: [],
+    all: function() { return this.products},
```

Comment:
You could rename it to _getAllCartProducts_ or _getAllProductsFromCart_ for better readability